process_pr.py

```diff
@@ -2081,6 +2084,16 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
         signatures["tests"] = "approved"
         labels.append("tests-" + override_tests_failure)
 
+    # automatically approve CATS_TO_APPROVE_ON_TEST on test-approved
+    if ("tests" in signatures) and (signatures["tests"] == "approved"):
```

Comment:
`if signatures.get("tests") == "approved":`

`get()` will return `None` if key is missing.